tests/std/tests/P0218R1_filesystem/test.cpp

```diff
@@ -3571,16 +3591,18 @@ void test_create_directory() {
     // test bogus path
     {
         error_code ec;
-        EXPECT(create_directory(nonexistentPath, ec) == false); // failed
-        EXPECT(bad(ec));
+        for (auto&& nonexistent : nonexistentPaths) {
+            EXPECT(create_directory(nonexistent, ec) == false); // failed
+            EXPECT(bad(ec));
 
-        EXPECT(throws_filesystem_error([] { create_directory(nonexistentPath); }, "create_directory", nonexistentPath));
+            EXPECT(throws_filesystem_error([&] { create_directory(nonexistent); }, "create_directory", nonexistent));
+        }
     }
 
     // test VSO-654638 where create_directory(p, existing_p) was doing copy_symlink behavior
     {
         error_code ec;
-        create_directory_symlink(nonexistentPath, p, ec);
+        create_directory_symlink(nonexistentPaths[0], p, ec);

```

Comment:
Is it critical that `nonexistentPaths[0]` be a local nonexistent path? If so, should that be commented above?

Reply:
Nope, it just needed to be any nonexistent path. Do you want a separate variable like "anyGivenNonexistentPath"? (I want to avoid reusing `nonexistentPath` when there's already a variable `nonexistentPaths`)

Reply:
No, I think it's fine.